compiler/modules/print.llvm/llvm.bal

```diff
@@ -242,18 +239,19 @@ public class Context {
         }
     }
 
-    function getStructName(StructType ty) returns [string, Type[]?]? {
-        foreach var entry in self.namedStructTypes.entries() {
-            var data = entry[1];
-            if data[0] === ty {
+    function getNamedStructBody(StructType ty) returns Type[] {
+        string? tyName = ty.name;
+        if tyName is string {
+            var data = self.namedStructTypes[tyName];
```

Comment:
You are over-using var.  If the type isn't too complicated, write the type not `var`: it makes the code easier to understand.